docs/notifications.rst

```diff
@@ -18,9 +18,16 @@ information.
 
 .. note::
    API Tokens
-      - Slack API Tokens are required to use this module. Reference their
-        `Authentication documentation <https://api.slack.com/docs/oauth>`_ for
-        additional information on authentication.
+      - Slack API Tokens are required to use this module. To obtain an API
+        Token, `create a Slack App <https://api.slack.com/apps>` associated
```

Comment:
``
`create a Slack App <https://api.slack.com/apps>`_
``

Missing the trailing underscore.